openthread/cmake/extensions.cmake

```diff
@@ -83,7 +83,14 @@ function(openthread_libs_configuration_write CONFIG_FILE)
     list(INSERT OPENTHREAD_SETTINGS 0 "OpenThread_commit=${git_describe}")
   endif()
 
-  FILE(WRITE ${CONFIG_FILE} ${OPENTHREAD_SETTINGS})
+  # Store compiler and Zephyr SDK version
```

Comment:
Note there's also `gcc -dumpfullversion` if only the version is needed (without the build details).

Reply:
Yes, but `--version` shows the Zephyr SDK version as well:
`arm-zephyr-eabi-gcc (Zephyr SDK 0.14.1) 10.3.0`

Reply:
Right, so that's indeed better.